src/pages/Landingpage.js

```diff
@@ -1,20 +1,15 @@
 import styled from 'styled-components';
 import { useNavigate } from 'react-router-dom';
-import catPictureBackground from '../images/cat_pic_450_700.svg';
-import headlinestyled from '../images/headingCatyummyStyled.svg';
+import catPictureBackground from '../images/backgroundimg.png';
+import { ReactComponent as Headlinestyled } from '../images/headingCatyummyStyled.svg';
 
 export default function LandingPage() {
   const navigate = useNavigate();
   return (
     <StyledLandingPage>
       <h1>
         <span className="sr-only">CATYUMMY</span>
-        <img
-          src={headlinestyled}
-          alt="headline CATYUMMY"
-          width="310px"
-          height="150px"
-        />
+        <Headlinestyled width="310px" height="150px" />
```

Comment:
still struggling with myself if i should work like this as well or just keep my svgs as import😅🤔

Reply:
i think the amount of code lines could be a good help for this decision